src/boot/idle.c

```diff
@@ -84,7 +89,5 @@ void Idle_ThreadEntry(void* arg) {
     osStartThread(&gMainThread);
     osSetThreadPri(NULL, OS_PRIORITY_IDLE);
 
-    while (1) {
-        ;
-    }
+    for (;;) {}
```

Comment:
is this stylistic or actual codegen difference

Reply:
Actual codegen difference, there's a different number of generated `nop`s. This was taken from MM